app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -204,12 +205,19 @@ class GithubWebhook extends RequestHandler<Body> {
           !file.filename.startsWith('dev/bots/')) {
         needsTests = true;
```

Comment:
`A PR that only affects comments (including documentation) is exempt from the need to add tests.` could be supported by looking at the file.patch string, assuming that's the diff for the change.
We'd need to check that every line that starts with a `+` or `-` matches the regexp `^[+-] *//`.

Reply:
This is a bit trickier I think. IIRC the diffs are only available in 100 line chunks, and we already flirt with the github api request limit, but I'll double check this.

Reply:
oh i didn't realize we didn't get the whole thing immediately.